tools/builder/react.js

```diff
@@ -0,0 +1,210 @@
+/**
+ * External dependencies
+ */
+import banner from 'gulp-banner';
+import fs from 'fs';
+import log from 'fancy-log';
+import gulp from 'gulp';
+import gulpif from 'gulp-if';
+import PluginError from 'plugin-error';
+import rename from 'gulp-rename';
+import saveLicense from 'uglify-save-license';
+import sourcemaps from 'gulp-sourcemaps';
+import tap from 'gulp-tap';
+import uglify from 'gulp-uglify';
+import webpack from 'webpack';
+
+function getWebpackConfig() {
+	return Object.create( require( './../../webpack.config.js' ) );
+}
+
+export const watch = function() {
+	const config = getWebpackConfig();
+
+	return webpack( config ).watch(
+		100,
+		onBuild.bind(
+			this,
+			function( error ) {
+				if ( error ) {
+					log( error );
+					return;
+				}
+				buildStatic( function() {} );
+			}
+		)
```

Comment:
Hmm, why the new callback format? The [old one](https://github.com/Automattic/jetpack/pull/9042/files#diff-ba210a9157252cc983268fdc3aa3ed52L218) was a bit easier on the eye :sweat_smile:

Reply:
If it's mainly to invoke `buildStatic` -- should that maybe then become a task of its own?